react-events/my-app/src/App.js

```diff
@@ -0,0 +1,53 @@
+import logo from './logo.svg';
+import './App.css';
+
+function CustomButton({ text, color, onCustomClick }) {
```

Comment:
let's extract out component definitions to a separate file

Reply:
Sorry! Just updated and will do that from now on.